Jenkinsfile

```diff
@@ -129,7 +129,11 @@ pipeline {
   stages {
 
     stage('build') {
-      agent any
+      agent {
+        kubernetes {
+          inheritFrom 'centos-8-6gb'
```

Comment:
This might be xtext-specific template, as it seems to be defined exclusively in https://github.com/eclipse-cbi/jiro/blob/49a71c613efd7cfe86ddefe34bbd72bedc603c31/instances/modeling.tmf.xtext/config.jsonnet#L43

Reply:
Btw, there was a note in our old k8s, I don't remember who added it, maybe me, maybe it was there even before. Labels are deprecated, we should use inheritFrom too. These settings were for the "root" machine, build machines had higher requests. I tested it to find out where is the maximum for limis, and then tried to find optimal shares for jobs. With additional resource packs perhaps those limits moved too.

```
      # fixes random failure: minimum cpu usage per Pod is 200m, but request is 100m.
      # affects performance on large repositories
      limits:
        memory: "1200Mi"
        cpu: "300m"
      requests:
        memory: "1200Mi"
        cpu: "300m"
```